llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -1176,6 +1176,7 @@ PPCTargetLowering::PPCTargetLowering(const PPCTargetMachine &TM,
       setTruncStoreAction(MVT::f128, MVT::f32, Expand);
 
       // No implementation for these ops for PowerPC.
+      setOperationAction(ISD::FSINCOS, MVT::f128, Expand);
```

Comment:
Have you checked other math related ISD nodes? Are MVT::f128 correctly handled for them?

Reply:
Yes, according to https://en.cppreference.com/w/c/numeric/math and nodes defined in ISDOpcodes.h. (they were already tested in f128-arith/round/fma.ll)